src/messageHandler.ts

```diff
@@ -1,45 +1,43 @@
-import { partnerEvents } from './events';
 import type {
-  ShareEvent,
-  IdTokenDidExpireEvent,
-  AccountProvisionRequestedEvent,
-  ExitRequestedEvent,
-  DreamsEvent,
-  InvestmentAccountProvisionRequestedEvent,
-  InvestmentAccountProvisionRequestedMessage,
-  InvestmentSellRequestedEvent,
-  InvestmentSellRequestedMessage,
+  DESEvent,
+  OnAccountProvisionRequestedEvent,
+  OnAccountRequestedEvent,
+  OnExitRequestedEvent,
+  OnIdTokenDidExpireEvent,
+  OnInvestmentAccountProvisionRequestedEvent,
+  OnInvestmentSellRequestedEvent,
+  OnShareEvent,
+  OnTransferConsentRequestedEvent,
+} from './des_events';
+
+import type {
+  AccountProvisionedEvent,
+  AccountProvisionedMessage,
+  InvestmentAccountProvisionInitiatedEvent,
+  InvestmentAccountProvisionInitiatedMessage,
+  NavigateToEvent,
+  OnAccountRequestedFailedEvent,
+  OnAccountRequestedFailedMessage,
+  OnAccountRequestedSucceededEvent,
+  OnAccountRequestedSucceededMessage,
+  OnTransferConsentCancelledEvent,
+  OnTransferConsentCancelledMessage,
+  OnTransferConsentSucceededEvent,
+  OnTransferConsentSucceededMessage,
   PartnerEvent,
   UpdateTokenEvent,
-  NavigateToEvent,
-  AccountProvisionInitiatedEvent,
-  InvestmentAccountProvisionInitiatedEvent,
-  Message,
   UpdateTokenMessage,
-  TransferConsentRequestedEvent,
-  TransferConsentRequestedMessage,
-  TransferConsentRequestSucceededEvent,
-  TransferConsentRequestCancelledEvent,
-  TransferConsentRequestCancelledMessage,
-  TransferConsentRequestSucceededMessage,
-  AccountRequestedEvent,
-  AccountRequestedFailedMessage,
-  AccountRequestedFailedEvent,
-  AccountRequestedSucceededEvent,
-  AccountRequestedSucceededMessage,
-} from './events';
+} from './partner_events';
 
 export type ClientCallbacks = {
-  onIdTokenDidExpire?: (event: IdTokenDidExpireEvent) => Promise<any>;
-  onAccountProvisionRequested?: (event: AccountProvisionRequestedEvent) => Promise<any>;
-  onInvestmentAccountProvisionRequested?: (
-    event: InvestmentAccountProvisionRequestedEvent,
-  ) => Promise<InvestmentAccountProvisionRequestedMessage>;
-  onInvestmentSellRequested?: (event: InvestmentSellRequestedEvent) => Promise<InvestmentSellRequestedMessage>;
-  onShare?: (event: ShareEvent) => Promise<any>;
-  onExitRequested: (event: ExitRequestedEvent) => Promise<any>;
-  onTransferConsentRequested?: (event: TransferConsentRequestedEvent) => Promise<TransferConsentRequestedMessage>;
-  onAccountRequested?: (event: AccountRequestedEvent) => Promise<any>;
```

Comment:
This was wrong. We expect an `OnAccountRequestedSucceededMessage` containing a `requestId`.